public/css/app.css

```diff
@@ -692,6 +696,11 @@ div.sprite {
     padding: 20px 24px;
   }
 
+  .mw-860 {
```

Comment:
I think these values may override the values on L164-165 above, and not sure if this should also be `.mw-600` or more likely, merged with the `.mw-600` (maybe) selector on L672-674.

Or maybe I'm way off base, since zooming out a bit on the diff shows that this is within a `@media screen and (max-width: 800px) {..}` media query.